.github/workflows/linux.yml

```diff
@@ -90,6 +95,52 @@ jobs:
           name: test-results
           path: '**/build/test-results/**/TEST-*.xml'
 
+  check-jre-packages:
```

Comment:
this is a lot of duplicate code, I'd suggest adding `image_type: [Jdk, Jre]` to the matrix and then going from there. Let me know if you need any help refactoring this

Reply:
In the course of the implementation it helped to address one distro after the other and always get a green build on GH actions. After I covered all distros now it can be simplified/deduplicated.